OpenRA.Mods.Common/Scripting/Properties/CombatProperties.cs

```diff
@@ -86,7 +86,10 @@ public void Attack(Actor targetActor, bool allowMove = true)
 			if (!targetActor.HasTrait<FrozenUnderFog>() && !Self.Owner.CanTargetActor(targetActor))
 				Log.Write("lua", "{1} is not revealed for player {0}!", Self.Owner, targetActor);
 
-			attackBase.AttackTarget(target, true, allowMove);
+			if (attackBase == null)
+				Log.Write("lua", "{0} does not have an Attack* trait. Ignoring order to attack {1}.".F(Self, targetActor));
```

Comment:
However there is no mean to check if the actor has Attack* traits in Lua so this will crash and the scripter can't fix it. The checks above also just log stuff.